OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameCashCounterLogic.cs

```diff
@@ -28,7 +28,6 @@ public class IngameCashCounterLogic : ChromeLogic
 		readonly PlayerResources playerResources;
 		readonly LabelWithTooltipWidget cashLabel;
 		readonly CachedTransform<(int Resources, int Capacity), string> siloUsageTooltipCache;
-		readonly string cashTemplate;
```

Comment:
Shouldn't we keep formatting?

Reply:
There is none:
https://github.com/OpenRA/OpenRA/blob/939f715e3c1cd82a080a26d57c29b5f049f6f4bb/mods/cnc/chrome/ingame.yaml#L1740